### DISCUSSION TITLE
v1.16.4

### DISCUSSION THREAD

**Discussion by olleolleolle** [github-changelog-generator/github-changelog-generator#968] [Announcements]
- #966 Feature: Allow named markdown links @kikito 
- #967: Fix: Directly handle rate limiting using `client.rate_limit.resets_in`. @ioquatix

<hr /><em>This discussion was created from the release <a href='https://github.com/github-changelog-generator/github-changelog-generator/releases/tag/v1.16.4'>v1.16.4</a>.</em>

**Comment by Kiko191** [comment]
gem 'rack', '~> 1.6'

**Reply by Kiko191** [reply]
[Uploading github-changelog-generator-master.zip…]()

**Reply by Kiko191** [reply]
gem 'rack', '~> 1.6'
